planning/behavior_path_planner/src/scene_module/pull_over/pull_over_module.cpp

```diff
@@ -1118,17 +1124,19 @@ void PullOverModule::decelerateForTurnSignal(const Pose & stop_pose, PathWithLan
   const Pose & current_pose = planner_data_->self_odometry->pose.pose;
 
   for (auto & point : path.points) {
-    const auto distance_to_stop = std::max(
+    const double distance_to_stop = std::max(
       0.0, calcSignedArcLength(path.points, point.point.pose.position, stop_pose.position));
     const float decel_vel =
       std::min(point.point.longitudinal_velocity_mps, static_cast<float>(distance_to_stop / time));
     const double distance_from_ego = calcSignedArcLengthFromEgo(path, stop_pose);
     const auto min_decel_distance = calcFeasibleDecelDistance(decel_vel);
-    if (!min_decel_distance) {
+
+    constexpr double eps_distance = 0.1;
```

Comment:
[imo] It is better to add some comments what these process mean.

Reply:
thansk added comments in https://github.com/autowarefoundation/autoware.universe/pull/3377/commits/8c54359fd68c2993825e043f6b88d2908ed5c1d3